Shell/Options.cpp

```diff
@@ -3577,6 +3616,10 @@ bool Options::complete(const Problem& prb) const
   if (_demodulationRedundancyCheck.actualValue == DemodulationRedundancyCheck::OFF) {
     return false;
   }
+  if (_demodulationRedundancyCheck.actualValue == DemodulationRedundancyCheck::ENCOMPASS &&
```

Comment:
This is the bit we want to remove when merging with Marton's proper fix (and thinking about schedules at the same time).